### PR TITLE
documents: add mail to contestant & lycee

### PR DESCRIPTION
**Pull request by pguenezan**

### CODE REVIEW

File: prologin/contest/models.py [hunk 1]
```diff
@@ -114,6 +114,10 @@ class Type(ChoiceEnum):
     class Meta:
         ordering = ('date_begin',)
 
+    @classmethod
+    def qualification_for_edition(cls, year: int):
+        return cls.objects.select_related('center').get(edition__year=year, type=cls.Type.qualification.value)
```
Comment (seirl) [inline prologin/contest/models.py:119]: - I know the file isn't PEP-8 compliant, but could we avoid adding more non compliant stuff?
- Why do you need to select_related('center')?

File: prologin/documents/templates/documents/courrier-lancement-candidat.tex [hunk 2]
```diff
@@ -0,0 +1,88 @@
+\documentclass[11pt]{article}
```
Comment (seirl) [inline prologin/documents/templates/documents/courrier-lancement-candidat.tex:1]: Why isn't this using prologin-letter.sty?

File: prologin/documents/templates/documents/courrier-lancement-lycee.tex [hunk 3]
```diff
@@ -0,0 +1,106 @@
+\documentclass[9pt]{article}
```
Comment (seirl) [inline prologin/documents/templates/documents/courrier-lancement-lycee.tex:1]: Why isn't this using prologin-letter.sty?

File: prologin/documents/views.py [hunk 4]
```diff
@@ -471,6 +472,32 @@ def get_extra_context(self):
...
 
 
+class LaunchMailToContestantView(BaseDocumentView):
+    template_name = 'documents/courrier-lancement-candidat.tex'
+    pdf_title = _("Prologin %(year)s: mail to contestant")
+    filename = pgettext_lazy("Document filename", "launch-mail-contestant-%(year)s")
+
+    def get_extra_context(self):
+        context = super().get_extra_context()
+        context['pres'] = team.models.TeamMember.objects.get(year=self.year, role_code=team.models.Role.president.name)
```
Comment (seirl) [inline prologin/documents/views.py:482]: 79cols

**Review by seirl — CHANGES_REQUESTED** [review]

**Comment by juli0z** [comment]
Migrated to https://gitlab.com/prologin/concours/site/-/merge_requests/122

**Pull request closed without merging**